docs/how_to/visualize_a_quantum_circuit.rst

```diff
@@ -0,0 +1,140 @@
+###########################
+Visualize a quantum circuit
+###########################
+
+This guide shows how to get a visual representation of a quantum circuit.
+
+Build the circuit
+=================
+
+The first step is to create the circuit.
+
+.. testcode::
+
+    from qiskit import QuantumCircuit
+
+    qc = QuantumCircuit(3, 3)
+    qc.h(range(3))
+    qc.cx(0, 1)
+    qc.measure(range(3), range(3))
+
+Visualize the circuit
+=====================
+
+There are three different ways to visualize a circuit. You can use
+
+* The ``print()`` function.
+* The :meth:`~.QuantumCircuit.draw` method.
+* The :func:`~.circuit_drawer` function.
+
+``print()``
+-----------
+
+If you call the ``print()`` function on a :class:`~.QuantumCircuit` object, you will get an `ASCII art version <https://en.wikipedia.org/wiki/ASCII_art>`_ of the circuit diagram.
+
+.. testcode::
+
+    print(qc)
+
+.. testoutput::
+    :options: +NORMALIZE_WHITESPACE
+
+         ┌───┐     ┌─┐   
+    q_0: ┤ H ├──■──┤M├───
+         ├───┤┌─┴─┐└╥┘┌─┐
+    q_1: ┤ H ├┤ X ├─╫─┤M├
+         ├───┤└┬─┬┘ ║ └╥┘
+    q_2: ┤ H ├─┤M├──╫──╫─
+         └───┘ └╥┘  ║  ║ 
+    c: 3/═══════╩═══╩══╩═
+                2   0  1 
+
+:meth:`~.QuantumCircuit.draw`
+---------------------------------------------
+
+You can also call the :meth:`~.QuantumCircuit.draw` method on a :class:`~.QuantumCircuit` object to visualize it. If you don't specify any arguments you will get a plain text representation of the circuit.
+
+.. code-block:: python
+
+    qc.draw()
+
+.. code-block:: text
+
+         ┌───┐     ┌─┐   
+    q_0: ┤ H ├──■──┤M├───
+         ├───┤┌─┴─┐└╥┘┌─┐
+    q_1: ┤ H ├┤ X ├─╫─┤M├
+         ├───┤└┬─┬┘ ║ └╥┘
+    q_2: ┤ H ├─┤M├──╫──╫─
+         └───┘ └╥┘  ║  ║ 
+    c: 3/═══════╩═══╩══╩═
+                2   0  1  
+
+However, if you change the ``output`` argument, you can get other different renderings. The available options for this argument are:
+
+* ``'text'``: renders the circuit with ASCII art. It's the default option.
+* ``'mpl'``: uses `matplotlib <https://matplotlib.org/>`_ to render the circuit.
+* ``'latex'``: uses :math:`\LaTeX` to render the circuit. It requires a full `LaTeX <https://latex.org/forum/>`_ distribution and the package ``pdflatex``.
+* ``'latex_source'``: outputs the :math:`\LaTeX` source code that creates the ``'latex'`` rendering of the circuit.
+
+Because this optional or keyword argument is actually the first of this method, one can type ``qc.draw(option)`` instead of ``qc.draw(output=option)``.
```

Comment:
```suggestion
Because this optional keyword argument is the first of this method, you can type ``qc.draw(option)`` instead of ``qc.draw(output=option)``
```